spec/p2p/reactor/README.md

```diff
@@ -2,225 +2,31 @@
 
 Reactor is the generic name for a component that employs the p2p communication layer.
 
-A component has to implement the [`Reactor`][reactor-interface] interface
-in order to use communication services provided by the p2p layer.
-This interface is currently the main source of documentation for a reactor.
+This section documents this **northbound interface** of the p2p layer,
+or the interaction of the p2p layer with reactors.
+The diagram below summarizes this interaction, representing some relevant event
+flows:
```

Comment:
Can we add a short sentence about the concurrency here? As far as I understand there are concurrent threads in the reactors and the p2p (for each connection). From these threads the arrows depict functions that are called on the other side of the interface.